docs/pages/build-reference/infrastructure.mdx

```diff
@@ -151,9 +153,9 @@ iOS worker VMs run on Mac Mini 8.1 hosts in an isolated environment. Every build
 - Hardware:
   - Intel: Intel(R) Core(TM) i7-8700B CPU (6 cores/12 threads), 64 GB RAM
   - M1: 3.2GHz 8-Core M1, 16GB RAM [2020]
-- Build resource limits:
-  - Intel: 3 cores, 12 GB RAM
-  - M1: 3 cores, 8 GB RAM
+- Build resources:
+  - [`intel-medium`](eas-json/#resourceclass-2): 3 cores, 12 GB RAM
+  - [`m1-medium`](eas-json/#resourceclass-2): 3 cores, 8 GB RAM
```

Comment:
is the number of cores here correct? Also what 3 cores in this context mean? M1 has performance and efficiency cores, so more detail can be useful in how resources are divided.

Reply:
I will take a look at it

Reply:
I found that our M1 hosts use the M1 processor with 8 cores (4 performance and 4 efficiency cores) and updated this information. I also noticed that actually, our M1 VMs use 2 cores vs 3 described above and updated it as well. 

Unfortunately, after quick research I wasn't able to find what using 2 cores per VM corresponds to in terms of access to performance and efficiency cores. Maybe @randomhajile has some ideas?